### PR TITLE
Upgrade to Marathon v1.1.1 and fix tests

### LINKED ISSUE #44: Marathon does not integrate in Mesos
I have used your kitchen test environment. All services come up and the tests are fine:
```
-----> serverspec installed (version 2.32.0)
       /opt/chef/embedded/bin/ruby -I/tmp/verifier/suites/serverspec -I/tmp/verifier/gems/gems/rspec-support-3.4.1/lib:/tmp/verifier/gems/gems/rspec-core-3.4.4/lib /opt/chef/embedded/bin/rspec --pattern /tmp/verifier/suites/serverspec/\*\*/\*_spec.rb --color --format documentation --default-path /tmp/verifier/suites/serverspec
       
       File "/opt/marathon/wrapper"
         should be file
       
       Service "zookeeper"
         should be running
       
       Service "mesos-master"
         should be running
       
       Service "mesos-slave"
         should be running
       
       Service "marathon"
         should be running
       
       Finished in 0.14531 seconds (files took 0.32164 seconds to load)
       5 examples, 0 failures
       
       Finished verifying <default-ubuntu-1404> (0m18.91s).
-----> Kitchen is finished. (0m19.47s)

```

But the marathon framework isn't active in mesos:

mesos_missing_marathon

And a simple /usr/bin/true task isn't scheduled:

marathon_app

### PR DESCRIPTION
**Pull request by rveznaver**
This should fix #44.

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -22,10 +23,11 @@ suites:
   - recipe[marathon::service]
   attributes:
     mesos:
-      version: 0.25.0
+      version: 0.28.1
     marathon:
       flags:
-        master: zk://localhost:2181/mesos
+        master: localhost:5050
```
Comment (pierrecdn) [inline .kitchen.yml]: So here you don't test zookeeper setup anymore. Is it expected ?

Comment (rveznaver) [inline .kitchen.yml]: Actually, the test was set up wrongly in the first place. As Mesos is by default set up without ZooKeeper, the ```--master``` flag has to point to the Mesos master, not ZooKeeper (this is why #44 was failing).
Unfortunately, due to https://github.com/mesosphere/marathon/issues/3606 it is still not possible to run Marathon without ZooKeeper, however that is set up by default to be ```zk://localhost:2181/marathon```, so we don't have to configure it.

Comment (pierrecdn) [inline .kitchen.yml]: OK, understood.

**Comment by pierrecdn** [comment]
+1

**Pull request merged**